app/controllers/ipaddresses_controller.rb

```diff
@@ -0,0 +1,108 @@
+class IpaddressesController < ApplicationController
+  include IpUtilities
+
+  def index
+    ip_address = ip_params['ip_addr']
+    ip_integer = ip_to_integer(ip_address)
+    invalid_ip_format(ip_address); return if performed?
+    private_invalid_ip(ip_address, ip_integer); return if performed?
```

Comment:
INFO: Style/Semicolon: Do not use semicolons to terminate expressions. (https://github.com/bbatsov/ruby-style-guide#no-semicolon)